### PR TITLE
Update add-a-printer-pop.md

### PR DESCRIPTION
**Pull request by Seibz**
added better instructions for their updated brother printer driver installer

### CODE REVIEW

File: content/add-a-printer-pop.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ sudo usermod -aG lpadmin $USER
 
 Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
 
-1. Click [this link](https://support.brother.com/g/b/faqend.aspx?c=us&lang=en&prod=hll2300d_us_eu_as&faqid=faq00100556_000) to download Brother's Driver Install Tool for Linux.
+1. Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for your appropriate printer.
```
Comment (ahoneybun) [inline content/add-a-printer-pop.md:48]: I believe this should be "for the appropriate printer"

**Review by ahoneybun — CHANGES_REQUESTED** [review]

File: content/add-a-printer-pop.md (hunk 1 above)
Comment (storyaddict) [reply content/add-a-printer-pop.md:48]: ☝️

Comment (Seibz) [reply content/add-a-printer-pop.md:48]: Will Do

**Review by Seibz — COMMENTED** [review]
looks good to me

**Review by storyaddict — APPROVED** [review]
Fixed wording issue.

**Review by ahoneybun — APPROVED** [review]

**Pull request merged**